river/sync.go

```diff
@@ -512,7 +513,11 @@ func (r *River) getFieldValue(col *schema.TableColumn, fieldType string, value i
 			v := reflect.ValueOf(value)
 			switch v.Kind() {
 			case reflect.Int, reflect.Int8, reflect.Int16, reflect.Int32, reflect.Int64:
-				fieldValue = r.makeReqColumnData(col, time.Unix(v.Int(), 0).Format(mysql.TimeFormat))
+				if v.Int() > math.MaxInt32 {
+					fieldValue = r.makeReqColumnData(col, time.Unix(0, int64(time.Millisecond)*v.Int()).Format(mysql.TimeFormat))
```

Comment:
any reference that we can know here using millisecond but not second?

Reply:
updated within README.md.
we saved create_time's millisecond timestamp as bigint in mysql.